src/components/footer/_footer.scss

```diff
@@ -44,7 +44,12 @@
   }
 
   .govuk-footer__link {
-    @include govuk-focusable-fill;
+    text-decoration: none;
```

Comment:
@dashouse are you happy for this change to be made in 'compatibility mode' as well?

Reply:
Good shout, will see what @dashouse thinks.